specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/searchindex.json

```diff
@@ -64,6 +64,325 @@
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name":"$orderby",
+            "in": "query",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "description": "The comma-separated list of OData $orderby expressions by which to sort the results. Each expression can be either a field name or a call to the geo.distance() function. Each expression can be followed by asc to indicate ascending, and desc to indicate descending. The default is ascending order. Ties will be broken by the match scores of documents. If no OrderBy is specified, the default sort order is descending by document match score. There can be at most 32 Orderby clauses.",
+            "x-ms-client-name": "OrderBy",
+            "x-ms-parameter-grouping": {
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name": "queryType",
+            "in": "query",
+            "type": "string",
+            "enum": [
+              "full",
+              "simple"
+            ],
+            "x-ms-enum": {
+             "name": "QueryType",
+             "modelAsString": false
+            },
+            "description": "Gets or sets a value that specifies the syntax of the search query. The default is 'simple'. Use 'full' if your query uses the Lucene query syntax.",
+            "x-ms-parameter-grouping": {
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name":"scoringParameter",
+            "in": "query",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "collectionFormat": "multi",
+            "x-ms-client-name": "ScoringParameterStrings",
+            "description": "The list of parameter values to be used in scoring functions (for example, referencePointParameter) using the format name:value. For example, if the scoring profile defines a function with a parameter called 'mylocation' the parameter string would be \"mylocation:-122.2,44.8\"(without the quotes).",
+            "x-ms-parameter-grouping": {
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name":"scoringProfile",
+            "in": "query",
+            "type": "string",
+            "description": "The name of a scoring profile to evaluate match scores for matching documents in order to sort the results.",
+            "x-ms-client-name": "ScoringProfile",
+            "x-ms-parameter-grouping": {
+              "name": "SearchParameters"
+            }
+          },
+          {
+            "name":"searchFields",
+            "in": "query",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "description": "The list of field names to include in the full-text search.",
+            "x-ms-client-name": "SearchFields",
```

Comment:
No need for x-ms-client-name if the name is the same (note that AutoRest should handle casing conventions automatically).

Reply:
Thanks- Removed. 

---
In reply to: [208102916](https://github.com/Azure/azure-rest-api-specs/pull/3578#discussion_r208102916) [](ancestors = 208102916)